dask/dataframe/io/parquet/core.py

```diff
@@ -711,6 +711,11 @@ def to_parquet(
             {"append": append, "compression": compression},
         )
     else:
+        if compute:
+            import dask
+
+            dask.compute(*part_tasks, **{compute_kwargs or {}})
```

Comment:
I think `part_tasks` is still just a list of keys. `dsk` is where the actual tasks are

Reply:
yeah, I'm iterating on this.  Stay tuned!

Reply:
Actually, see recent commit I think

Reply:
And the edit in the top comment

Reply:
👍